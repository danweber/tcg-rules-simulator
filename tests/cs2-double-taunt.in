VERSION:2
FIELD2:Atk1,CS2-04 Atk2,CS2-04
HAND2:CS2-05
DECK2:CS2-02
SECURITY1:CS1-02 CS1-03
GAME:P1 T1 MAIN 3
DUMP:SECURITY

CMD:PASS
CMD:MAIN
# We must pick which to do first
STEP:AUTOANSWER:CAPTURE
DUMP:SECURITY
EXIT


