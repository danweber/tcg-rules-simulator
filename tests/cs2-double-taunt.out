FIELD1:
FIELD2:Atk1,CS2-Rabiddog Atk2,CS2-Rabiddog
SECURITY1:CS1-Papershield CS1-Garbagecanlid
SECURITY2:
GAME:P1 T1 MAIN 3
CAPTURE:Rabid Dog [Start of Your Main Phase] Attack with this Monster.
CAPTURE:Rabid Dog [Start of Your Main Phase] Attack with this Monster.
FIELD1:
FIELD2:Atk1,CS2-Rabiddog,REST Atk2,CS2-Rabiddog
SECURITY1:CS1-Papershield
SECURITY2:
GAME:P2 T2 MAIN 3
